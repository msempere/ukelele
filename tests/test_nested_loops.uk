def main() void{
    int a;
    a = 5;
    println(a);
    do{
        int b;
        println(b);
        a = a + 1;
        for(int i=5; i>0; i=i-2)
        {
            int c;
            print(c);
            print(a);
            println(b);
        }
    }
    while(a<2)
    println(a);
    return();
}
