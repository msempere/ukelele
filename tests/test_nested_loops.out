5
0
060
060
060
6
